<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenDialog</class>
 <widget class="QDialog" name="OpenDialog">
  <property name="windowTitle">
   <string>Opening options</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="replaceButton">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="appendButton">
       <property name="text">
        <string>Append</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="folderCheckBox">
     <property name="text">
      <string>Open whole folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="rememberCheckBox">
     <property name="text">
      <string>Rembember my choice</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
